Riot/Assets/en.lproj/Vector.strings

```diff
@@ -1715,6 +1715,8 @@ Tap the + to start adding people.";
 // MARK: File upload
 "file_upload_error_title" = "File upload";
 "file_upload_error_unsupported_file_type_message" = "File type not supported.";
+"file_upload_error_too_large_title" = "File too large";
+"file_upload_error_too_large_message" = "Maximum supported file size is %@MB";
```

Comment:
Have you checked what this looks like? Now that there's an `NSByteCountFormatter` feeding the alert, it should already contain the appropriate unit formatted appropriately in the string.